internal/stats/main.go

```diff
@@ -0,0 +1,50 @@
+package stats
+
+import (
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	addr = flag.String("prometheus-listen-address", ":8080", "The address to listen on for HTTP requests.")
```

Comment:
Should this come from a config or environment var, instead of the commandline?

Reply:
I'd think a config, but this was the path of least resistance for this project since at the moment its all cli/flag passed